spec/rubocop/ast/traversal_spec.rb

```diff
@@ -64,6 +64,30 @@ def baz(&block)
     end
   end
 
+  context 'when a class defines `on_itblock`', :ruby34, broken_on: :parser do
+    let(:klass) do
+      Class.new do
+        attr_reader :calls
+
+        include RuboCop::AST::Traversal
+        def on_itblock(node)
+          (@calls ||= []) << node.children.first&.type
+          super
+        end
+      end
+    end
+
+    let(:source) { <<~RUBY }
+      foo { bar(it) }
+      xxx { _1 }
+      yyy { |zzz| }
```

Comment:
This example might be slightly adjustable. e.g.,:

```suggestion
      it_block { do_something(it) }
      numbered_block { do_something(_1) }
      normal_block { |arg| do_something(arg) }
```